compiler/src/iree/compiler/Codegen/Common/TransformExtensions/CommonExtensions.cpp

```diff
@@ -916,5 +917,75 @@ void transform_dialect::WorkgroupSwizzleOp::getEffects(
   transform::modifiesPayload(effects);
 }
 
+//===----------------------------------------------------------------------===//
+// TestVectorLayoutAnalysisOp
+//===----------------------------------------------------------------------===//
+
+static void setAnchorOpsFromAttributes(VectorLayoutAnalysis &analysis,
+                                       func::FuncOp funcOp) {
+  for (auto &block : funcOp) {
+    for (auto &op : block) {
+      for (auto attr : op.getAttrs()) {
+        std::string name = attr.getName().str();
+        if (name.find("__vector_layout_test_anchor_operand_") !=
```

Comment:
Why convert to string rather than using StringRef helpers?

Reply:
I couldn't find a way to use std::stoi on StringRef results. I will try to make it work with StringRef again.